bids/variables/entities.py

```diff
@@ -84,6 +84,7 @@ class NodeIndex(Node):
     """Represents the top level in a BIDS hierarchy. """
 
     def __init__(self):
+        super().__init__("top_of_bids_heirarchy", None)
```

Comment:
I'm not actually sure there's a reason for `NodeIndex` to inherit from `Node`. That might be vestigial. Maybe try changint to `object` and see if tests still pass. If we do keep the inheritance, let's set the first argument to super to `None`, in the event there's some place in the codebase that checks against `level` being `None` (I doubt it, but the string values here constitute a controlled vocabulary, and are not just generic descriptors, so `None` makes more sense anyway).